libr/include/r_util/pj.h

```diff
@@ -28,13 +28,15 @@ R_API PJ *pj_o(PJ *j);
 R_API PJ *pj_a(PJ *j);
 /* keys, values */
 R_API PJ *pj_k(PJ *j, const char *k);
+R_API PJ *pj_knull(PJ *j, const char *k);
```

Comment:
i dont like to use "null" here wwhen all the other cases have 1 letter to define the type.

Reply:
propose a better name